### PR TITLE
Updated drainage class options to be pulled in from API

### PR DESCRIPTION
**Pull request by mikahpinegar**
- Use new soil drainage API to get regional soil drainages

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.216.187.201:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.216.187.201:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/Location/SoilCondition/SoilDrainage/SoilDrainage.js [hunk 1]
```diff
@@ -54,11 +53,10 @@ const SoilDrainage = () => {
   }, [soilDataOriginalRedux]);
 
   useEffect(() => {
-    const checkArray = ['Very poorly drained', 'Poorly drained', 'Somewhat poorly drained'];
-    if (checkArray.some((e) => soilDataRedux?.drainageClass.includes(e))) {
+    if (drainageOptions.map((option) => option.value).indexOf(soilDataRedux?.drainageClass[0]) < 3) {
```
Comment (YazidZidane) [inline src/pages/Location/SoilCondition/SoilDrainage/SoilDrainage.js:56]: This is also showing tile drainage when there's not a drainage class value

**Review by YazidZidane — APPROVED** [review]

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.216.187.201:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,48.216.187.201:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Pull request merged**